build/autoloaderchecker.sh

```diff
@@ -22,13 +22,17 @@ echo "Regenerating main autoloader"
 $COMPOSER_COMMAND dump-autoload -d $REPODIR
 
 for app in ${REPODIR}/apps/*; do
+	git check-ignore ${app} -q
+	if git check-ignore ${app} -q ; then
+		echo
+		echo "${app} is not shipped. Ignoring autoloader regeneration:w"
```

Comment:
```suggestion
		echo "${app} is not shipped. Ignoring autoloader regeneration"
```

Trying to close an editor or something?